api/src/routes/identity/diff/get.ts

```diff
@@ -61,6 +69,53 @@ async function resolveDiff(
         }
         const client = identity.Client.fromConfig(config);
 
+        const resolvedDocument = identity.ResolvedDocument.fromJSON(document);
+
+        const receipt = await client.resolveDiffHistory(resolvedDocument);
+
+        const receiptObj = receipt.toJSON();
+
+        const diffChainData = [];
+
+        const chainData = receipt.chainData();
+
+        for (let i = 0; i < chainData.length; i++) {
+            resolvedDocument.mergeDiffMessage(chainData[i]);
+
+            const integrationMessage = {
+                message: receiptObj.chainData[i],
+                document: resolvedDocument.document.toJSON(),
+                messageId: chainData[i].messageId
+            };
+            diffChainData.push(integrationMessage);
+        }
+
+        return { chainData: diffChainData, spam: receiptObj.spam };
+    } catch (e) {
+        return { error: e as string };
```

Comment:
Do you actually know `e` is a string, is this a dangerous cast

Reply:
this is not new behavior unfortunately, the proper solution is to typeguard and stringify if `e` is no string. Will build a solution for that.

Reply:
decided to go with a quick `typeof` / `toString` solution. Does that work for you?

Reply:
Yes that should be sufficient